pkg/arvo/lib/chat-json.hoon

```diff
@@ -39,7 +40,7 @@
   =/  res  ((ot output+(ar dank) ~) a)
   ?^  res
     u.res
-  p:(mule |.([(sell (slap !>(..^zuse) (ream u.exp)))]~))  ::TODO  oldz
+  (eval:chat-eval *bowl:gall (ream u.exp))
```

Comment:
This is an interesting way to achieve determinism wrt to identity, time, and entropy ... ;)

I think that the proper approach would be to parse into an unevaluated-hoon message type, and then replace that with an evaluated-hoon message in the app. That would also be a further safeguard against accidentally eval-ing foreign hoons (although, barring scry, that's not really a safety issue; we only ever print the result).